examples/cors/cors_middleware.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import "github.com/savsgio/atreugo/v10"
+
+// CORS Setting
+var (
+	corsAllowHeaders     = "Content-Type, authorization"
+	corsAllowMethods     = "HEAD,GET,POST,PUT,DELETE,OPTIONS"
+	corsAllowOrigin      = "*"
+	corsAllowCredentials = "true"
+)
+
+func corsMiddleware(ctx *atreugo.RequestCtx) error {
+	// Avoid CORS middleware when you need
+	path := string(ctx.Path())
+	ignoredURL := map[string]bool{
+		"/no-cors": true,
+	}
+	if ignoredURL[path] {
```

Comment:
It's not needed because you could skip middlewares in groups/views

Reply:
Get it.